coordinator/tests/services/validation/bundle-validator.spec.ts

```diff
@@ -300,3 +301,85 @@ describe("verifyCancellationBundle", () => {
     expect(returnedErrors[0].expression).toContainEqual("Bundle.entry.resource.ofType(MedicationRequest).statusReason")
   })
 })
+
+describe("verifyDispenseNotificationBundle", () => {
+  let bundle: fhir.Bundle
+
+  beforeEach(() => {
+    const dispenseExample = TestResources.specification.map(s => s.fhirMessageDispense).filter(isTruthy)[0]
+    bundle = clone(dispenseExample)
+  })
+
+  test("accepts a valid dispense request", () => {
+    const returnedErrors = validator.verifyDispenseBundle(bundle)
+    expect(returnedErrors.length).toBe(0)
+  })
+
+  test("returns an error when MedicationDispenses have different prescription statuses", () => {
+    const medicationDispenseEntry =
```

Comment:
can use `getMedicationDispenses`?

Reply:
Not quite, the entry includes the top level, not just the resource itself